src/codeu/chat/client/core/Controller.java

```diff
@@ -112,4 +114,27 @@ public ConversationHeader newConversation(String title, Uuid owner)  {
 
     return response;
   }
+  
+  public Time statusUpdate() {
+	Time updateTime = null;
+	try (final Connection connection = source.connect()) {
+
+	  Serializers.INTEGER.write(connection.out(), NetworkCode.STATUS_UPDATE_REQUEST);
+	  
```

Comment:
Actually on second thoughts, the code will work in a single session, but if we were to save the interests between sessions via json, we would need to find a way to update the convo interests in the model.